cves/CVE-2015-5351.yml

```diff
@@ -43,15 +43,35 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
-
+description: |
+    Apache Tomcat includes a web application called the Manager. This tool is 
+    used to deploy new web applications, to undeploy currently deployed web 
+    applications, to reload existing web applications, and to perform other 
+    operations to change and monitor the availability of the user's web 
+    applications.
+    
+    Another web application included in Apache Tomcat is the Host Manager. The 
+    Host Manager is used to manage the different virtual hosts (server-sharing 
+    wesites) on the user's Tomcat server.
+    
+    CVE-2015-5351 refers to a vulnerability in the Manager and Host Manager web 
+    applications. An unauthenticated user attempting to accessing the index page 
+    of either of these web applications would be issues a redirect from the 
+    server with a valid CSRF token. An attacker may use this token to send 
```

Comment:
"An unauthenticated user attempting to accessing the index page of either of these web applications would be issues a redirect from the server with a valid CSRF token. "
I had trouble understanding this sentence here, consider proof reading this bit over again. It seems that there was a missing piece here.